Gemfile

```diff
@@ -55,6 +55,7 @@ gem 'bootsnap', require: false
 # gem "image_processing", "~> 1.2"
 
 group :development, :test do
+  gem 'database_cleaner'
```

Comment:
- [ ] the Capybara gem should be included for the integration tests

Reply:
@mikethreels, Thanks for clearing that out. But adding the Capybara gem in the `group :test`, is that not sufficient?
**Here:** https://github.com/Yazino12/blog-app/blob/2c488e6412024d0637bf3c9cd64f343f65cc4782/Gemfile#L73-L78

Reply:
@mikethreels for the n+1 problem in posts#index when fetching all posts and their comments: I structured the code in a way that it uses the posts recent_comments method. and the problem does not exist in this case. So if I add the `includes` it will alert me to remove it.
**If you clone the code locally and try it you won't get any alerts from the bullet.**

Reply:
Hi @Yazino12,
The gem should be in the development, test environment

Reply:
@mikethreels, Thanks for the response, I will add that now 😊

Reply:
@Yazino12,
Regarding the N+1 problem, I'm not at my computer right now so I'm not able to check the exact requirements, However the way you specify it you're only getting the most recent comments instead of all the comments when needed. But like I mentioned before I'm not sure. If you don't agree with that requirement please explain why in the PR and request for another review.

Reply:
@mikethreels, Thanks for that clarification. So for **posts_index** page I need to show only the **recent_comments** which I can do that by `@post.recent_comments`, and remember in the **Posts_controller** `index` method I am accessing the **@post** NOT THE **@posts**. Only in the posts_show page I need to show all the **comments** for a user.